@uportal/esco-content-menu/src/services/sizeTools.js

```diff
@@ -0,0 +1,17 @@
+export default {
+  elementWidth(ref) {
```

Comment:
:art: these may be able to be directly exported

```js
export function elementWidth(ref) {/*...*/}
export function breakPointName (size) {/*...*/}

Reply:
I didn't know which form was the best and so I goes on this one. But I will change !